src/device.js

```diff
@@ -77,7 +77,31 @@ var devicePixelRatio = window.originalDevicePixelRatio || window.devicePixelRati
 // @deprecated
 exports.devicePixelRatio = devicePixelRatio;
 
-exports.screen.devicePixelRatio = devicePixelRatio;
+var ctx = document.createElement('Canvas').getContext('2d');
+
+// Provide a pixel ratio for our canvas
+Object.defineProperty(exports.screen, 'devicePixelRatio', {
+	get: function() {
+		if (exports.isSimulator) {
+			return 1;
```

Comment:
It can be, but I disabled scaling when in simulator because it would break the view inspector.